src/main/java/de/retest/recheck/ui/descriptors/Element.java

```diff
@@ -213,6 +213,21 @@ public Attributes getAttributes() {
 		return attributes;
 	}
 
+	/**
+	 * Returns the value of the given attribute, whether this is an identifying attribute or not.
+	 *
+	 * @return The {@link java.io.Serializable} value of the attribute.
+	 */
+	public Object getAttributeValue( final String attributeName ) {
```

Comment:
The implementation "prefers" identifying over ordinary attributes. It might be that we have the same key/name in both sets, for example `type` in recheck-web. Here, the identifying attribute is the tag name, whereas the ordinary attribute refers to the corresponding [HTML attribute](https://www.w3schools.com/tags/att_type.asp).

While the implementation is fine, I would suggest to add a test for this behavior (e.g. `getAttributeValue_should_prefer_identifying_over_ordinary_attribute`) in case someone accidentially changes this.

Reply:
Done